mp4parse_capi/src/lib.rs

```diff
@@ -895,6 +897,14 @@ fn get_track_audio_info(
             }
             AudioCodecSpecific::MP3 => Mp4parseCodec::Mp3,
             AudioCodecSpecific::ALACSpecificBox(_) => Mp4parseCodec::Alac,
+            AudioCodecSpecific::AMRSpecificBox(_) | AudioCodecSpecific::AMRSpecificEmptyBox
+                if audio.codec_type == CodecType::AMRNB =>
+            {
+                Mp4parseCodec::AMRNB
+            }
+            AudioCodecSpecific::AMRSpecificBox(_) | AudioCodecSpecific::AMRSpecificEmptyBox => {
```

Comment:
The origin intent was for the case that AudioCodecSpecific::AMRSpecificBox doesn't exist but AudioCodecSpecific::AMRSpecificEmptyBox.

But since it's better to use AMRSpecificBox with an empty TryVec to replace AMRSpecificEmptyBox.
So this part could be simplified as your commit  4a81b3a.

Thanks.

Reply:
Yes, your way is easier to read, I'll use yours, thanks.